pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -692,20 +697,21 @@ func (m *kubeGenericRuntimeManager) purgeInitContainers(pod *v1.Pod, podStatus *
 }
 
 // findNextInitContainerToRun returns the status of the last failed container, the
-// next init container to start, or done if there are no further init containers.
+// index of next init container to start, or done if there are no further init containers.
```

Comment:
Why changing this to index?

Reply:
Sorry, I remember there was a reason... But I can't remember it now... Will change it back.

Reply:
Done